images/kubekins-e2e/variants.yaml

```diff
@@ -3,8 +3,8 @@ variants:
     CONFIG: experimental
     GO_VERSION: 1.15
     K8S_RELEASE: stable
-    BAZEL_VERSION: 2.2.0
-    OLD_BAZEL_VERSION: 0.28.1
+    BAZEL_VERSION: 3.4.1
+    OLD_BAZEL_VERSION: 2.2.0
```

Comment:
we should probably update the master key below if we're updating images/bazel/variants for the kubernetes-master config above?

Reply:
Ah ya, fixed.